### ISSUE TITLE
version of fairseq

### ISSUE THREAD

**Issue by cedar33** [travel-go/Abstractive-Text-Summarization#4]
when I run `train.py`, an error occured and said 'no module named data', then I copied the `data` module from pytorch's fairseq(version 0.8), more errors come out, which fairseq`s version in your code?

**Comment by travel-go** [comment]
The fairseq's version is 0.5.0 and I have uploaded the data module.

**Comment by cedar33** [comment]
> The fairseq's version is 0.5.0 and I have uploaded the data module.

Thank you, the code works well,  but I got some confused result like this
`| epoch 001:   3%| | 322/11899 [02:29<1:28:07,  2.19it/s, loss=1820.630, ppl=inf, wps=8635, ups=2.0, wpb=3873, bsz=203, num_updates=322, lr=0.25, gnorm=23.282, clip=2%
| epoch 001:   3%| | 322/11899 [02:24<1:28:08,  2.19it/s, loss=1820.630, ppl=inf, wps=8617, ups=2.0, wpb=3873, bsz=203, num_updates=322, lr=0.25, gnorm=23.282, clip=2%
| epoch 001:   3%| | 323/11899 [02:25<1:26:44,  2.22it/s, loss=1826.219, ppl=inf, wps=8614, ups=2.0, wpb=3871, bsz=203, num_updates=323, lr=0.25, gnorm=23.288, clip=2%
| epoch 001:   3%| | 323/11899 [02:30<1:26:45,  2.22it/s, loss=1826.219, ppl=inf, wps=8632, ups=2.0, wpb=3871, bsz=203, num_updates=323, lr=0.25, gnorm=23.288, clip=2%
| epoch 001:   3%| | 324/11899 [02:30<1:25:54,  2.25it/s, loss=1830.204, ppl=inf, wps=8625, ups=2.0, wpb=3868, bsz=203, num_updates=324, lr=0.25, gnorm=23.293, clip=2%
| epoch 001:   3%| | 324/11899 [02:25<1:25:58,  2.24it/s, loss=1830.204, ppl=inf, wps=8608, ups=2.0, wpb=3868, bsz=203, num_updates=324, lr=0.25, gnorm=23.293, clip=2%
| epoch 001:   3%| | 325/11899 [02:26<1:25:48,  2.25it/s, loss=1834.157, ppl=inf, wps=8606, ups=2.0, wpb=3867, bsz=203, num_updates=325, lr=0.25, gnorm=23.298, clip=2%
| epoch 001:   3%| | 325/11899 [02:30<1:25:50,  2.25it/s, loss=1834.157, ppl=inf, wps=8624, ups=2.0, wpb=3867, bsz=203, num_updates=325, lr=0.25, gnorm=23.298, clip=2%|`
the loss was increasing. I`m trying to debug the code, and it will be appreciate if you can give some advice

**Comment by cedar33** [comment]
> The fairseq's version is 0.5.0 and I have uploaded the data module. 

thanks, I have get this after 8h training, however the summary is unsatisfactory.the dataset is LCSTS dataset.
`| epoch 006: 100%|▉| 11890/11899 [1:20:56<00:03,  2.39it/s, loss=6.979, ppl=126.17, wps=9312, ups=2.4, wpb=3803, bsz=202, num_updates=71385, lr=0.00025`
